projects/1.21/assets/advancedae/advanced_ae/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "block.advanced_ae.adv_pattern_provider": "ME高级扩展样板供应器",
+  "block.advanced_ae.data_entangler": "量子数据纠缠器",
+  "block.advanced_ae.quantum_accelerator": "量子计算机加速器",
+  "block.advanced_ae.quantum_core": "量子计算机核心",
+  "block.advanced_ae.quantum_multi_threader": "量子计算机多线程处理器",
+  "block.advanced_ae.quantum_storage_128": "128M量子计算机合成存储器",
+  "block.advanced_ae.quantum_storage_256": "256M量子计算机合成存储器",
+  "block.advanced_ae.quantum_structure": "量子计算机结构外壳",
+  "block.advanced_ae.quantum_unit": "量子合成单元",
+  "block.advanced_ae.reaction_chamber": "反应室",
+  "block.advanced_ae.small_adv_pattern_provider": "ME高级样板供应器",
+  "emi.category.advanced_ae.EmiReactionChamber": "反应室",
+  "emi.text.advanced_ae.ReactionChamberEnergy": "能量使用: %dk FE",
```

```suggestion
  "emi.text.advanced_ae.ReactionChamberEnergy": "能量使用：%dk FE",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。